sw/device/tests/BUILD

```diff
@@ -1314,24 +1314,23 @@ opentitan_test(
     ],
 )
 
-opentitan_functest(
+opentitan_test(
     name = "gpio_pinmux_test",
     srcs = ["gpio_pinmux_test.c"],
-    cw310 = cw310_params(
-        interface = "hyper310",
-        test_cmds = [
-            "--bitstream=\"$(location {bitstream})\"",
-            "--bootstrap=\"$(location {flash})\"",
-        ],
+    exec_env = {
+        "//hw/top_earlgrey:fpga_hyper310_test_rom": None,
+        "//hw/top_earlgrey:sim_verilator": None,
+    },
+    hyper310 = new_cw310_params(
```

Comment:
Because of `new_cw310_params()` here, this is still tagged with "cw310", making hyper310 tests potentially not distinct from cw310 tests.

It's possible it's not a problem, though. :thinking: 

Is it enough to use this rule? (though it _is_ kind of awkward, hehe)
Targeting hyper310: `--test_tag_filters=hyper310`
Targeting cw310: `--test_tag_filters=cw310,-hyper310`

Reply:
Oh, right, but the "hyper310" tag isn't actually applied, in this case.